test/representers/api/paged_collection_representer_test.rb

```diff
@@ -52,4 +38,22 @@
     representer.href_url_helper({foo: 1, bar: 2, camp: 3}).must_equal "this is a test"
   end
 
+  describe "requires routes" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.